src/module/item/sheet/item-sheet.js

```diff
@@ -554,35 +554,34 @@ export default class ZweihanderItemSheet extends ItemSheet {
 
     if (item.type !== 'ancestry') return;
 
-    const worldTable = game.tables.find((table) => table.name.includes(item.name));
```

Comment:
Please revert this change. The randomization of Ancestral Traits is meant to use World Tables first, and only use Compendium as a backup, otherwise it becomes extremely finicky to play with homebrew Ancestral Traits, e.g. if I have a custom Ancestry "Human (Northerner)," the current implementation in this PR will just not work, even if there is a World Table named "Human (Northerner)."

Reply:
Reverted last commit